src/views/RecIM/views/Activity/components/ScheduleList/index.jsx

```diff
@@ -99,13 +115,40 @@ const ScheduleList = ({
         case 'Single Elim':
           return numTeams - 1;
         case 'Ladder':
-          return 1; //temporary
+          return '1 (by default)'; //temporary
         case 'Double Elim':
           return numTeams * 2 - 1;
         default:
           return null;
       }
     };
+
+    let parameterFields = (type, numTeams) => {
+      return (
+        <Box component="form">
+          <FormControl>
```

Comment:
What are the `Box` and `FormControl` for? You aren't using any of the features of the `form` component. It seems like just the `TextField` will give you exactly the same result.